guards/index.js

```diff
@@ -0,0 +1,9 @@
+import { requireActiveSession } from "./requireActiveSession.js";
+import { requireNoActiveSession } from "./requireNoActiveSession.js";
+
+export const guardMap = {
+  activeSession: requireActiveSession,
+  noSession: requireNoActiveSession,
+};
+
+export { requireActiveSession, requireNoActiveSession };
```

Comment:
Why renaming them and double exporting? I think the original names are better suited and self explanatory of what they guard for.
```suggestion
export * from "./requireActiveSession.js";
export * from "./requireNoActiveSession.js";
```

Then usage in commands becomes
```js
import { requireSession, requireBacon } from "../../guards/index.js";

const guardArr = [requireSession, requireBacon];
```
Or
```js

import * as guards from "../../guards/index.js";

const { requireSession, requireBacon } = guards;
const guardArr = [requireSession, requireBacon];
```